test/lib/jasmine-execute.js

```diff
@@ -16,4 +16,31 @@ document.write('<style> @import \'../vendor/jasmine/jasmine.css?_=' + (+new Date
   window.onload = function() {
     env.execute();
   };
+
+  var oldError = console.error;
+  var newError = function() {
+    oldError.apply(this, arguments);
+    var spec = env.currentSpec;
+    if (spec) {
+      var expectationResult = new jasmine.ExpectationResult({
+        passed: false,
+        message:
+          'Expected test not to warn. If the warning is expected, mock it ' +
+          'out and test that the warning occurs.',
```

Comment:
I know. It's easy to copy/paste this to the jest config but I figured it's better to just have it in one place for now, so I opted for here so Travis gets it.